pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -37,7 +37,7 @@ import (
 
 const (
 	// emptyUniquePodName is a UniquePodName for empty string.
-	emptyUniquePodName types.UniquePodName = types.UniquePodName("")
+	EmptyUniquePodName types.UniquePodName = types.UniquePodName("")
 
 	// emptyUniqueVolumeName is a UniqueVolumeName for empty string
 	emptyUniqueVolumeName v1.UniqueVolumeName = v1.UniqueVolumeName("")
```

Comment:
Actually I am not sure why UniquePodName is types.UniquePodName and UniqueVolumeName is v1.UniqueVolumeName @saad-ali ?